lib/Tokenizer.js

```diff
@@ -144,10 +144,13 @@ function Tokenizer(options, cbs){
 	this._ended = false;
 	this._xmlMode = !!(options && options.xmlMode);
 	this._decodeEntities = !!(options && options.decodeEntities);
+	this._isMarkdownCode = false;
+	this._isInequality = false;
 }
 
 Tokenizer.prototype._stateText = function(c){
-	if(c === "<"){
+	// Parse open tag if it is not Markdown and not part of an inequality
+	if(c === "<" && !this._isMarkdownCode && !this._isInequality){
```

Comment:
Why is `&& !this._isInequality` necessary?

Reply:
This is such that the Tokenizer doesn't think that the `<` of a `<=` is the start of an open HTML tag.